WordleGame/WordleGame/Views/Letter/LetterView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LetterView.swift
+//  WordleGame
+//
+//  Created by Alex on 3.05.22.
+//
+
+import Foundation
+import UIKit
+
+class LetterView: UIStackView {
```

Comment:
Is this the view that shows all words? If so, it's not clear from the name. You can call it `WordsContainerView`, `GameFieldView` and so on